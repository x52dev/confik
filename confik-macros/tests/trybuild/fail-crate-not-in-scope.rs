#[derive(confik::Configuration)]
#[confik(crate = not_in_scope)]
struct Config {}

fn main() {}
