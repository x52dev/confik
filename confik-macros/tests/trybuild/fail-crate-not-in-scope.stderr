error[E0433]: failed to resolve: use of undeclared crate or module `not_in_scope`
 --> tests/trybuild/fail-crate-not-in-scope.rs:2:18
  |
2 | #[confik(crate = not_in_scope)]
  |                  ^^^^^^^^^^^^ use of undeclared crate or module `not_in_scope`

error: cannot find attribute `serde` in this scope
 --> tests/trybuild/fail-crate-not-in-scope.rs:3:8
  |
3 | struct Config {}
  |        ^^^^^^
  |
  = note: `serde` is in scope, but it is a crate, not an attribute
